ci/e2e-tests/test-run.sh

```diff
@@ -880,8 +880,10 @@ ssh -i "$PWD/vm-ssh-key" "aziot@$vm_public_ip" "
         exit 1
     fi
 
-    device_twin=\"\$(~/iothub-get-twin.sh \"\$device_identity\")\"
-    printf 'Device twin: %s\n' \"\$device_twin\" >&2
+    if [ \"$OS\" != 'ubuntu:22.04' ] || [ \"$TEST_NAME\" != *'-x509'* ]; then
```

Comment:
You already have `$test_name`. You don't need a new env var.

Reply:
Oh right. Fixed.